charts/pomerium/templates/configmap.yaml

```diff
@@ -1,4 +1,5 @@
 {{- if not .Values.config.existingConfig }}
+{{- if .Values.operator.enabled }}
```

Comment:
This looks off. We'd lose these annotations in non operator mode right?

Reply:
The logic here is a little odd.  In operator mode we have two config maps - one empty, and one populated from the static values of the chart.  When running without the operator, we only need the main config map with static values.

To save a big repeated block in the conditional, I am changing the name of the ConfigMap with the static values depending on whether we're in operator mode.  Similarly, I just flip the empty ConfigMap off in non-operator mode.

Hopefully that tracks what's in the yaml...unless I'm missing something else.

Reply:
Makes  complete sense. 👍